services/cloudfront/src/main/java/software/amazon/awssdk/services/cloudfront/CloudFrontUtilities.java

```diff
@@ -75,9 +75,27 @@ public static CloudFrontUtilities create() {
      * This is a convenience which creates an instance of the {@link CannedSignerRequest.Builder} avoiding the need to
      * create one manually via {@link CannedSignerRequest#builder()}
      *
-     * @param request A {@link Consumer} that will call methods on {@link CannedSignerRequest.Builder} to create a request.
+     * @param request
+     *            A {@link Consumer} that will call methods on {@link CannedSignerRequest.Builder} to create a request.
      * @return A signed URL that will permit access to a specific distribution
      *         and S3 object.
+     *
+     * <p><b>Example Usage</b>
+     * <p>
+     * {@snippet :
+     *     CloudFrontUtilities utilities = CloudFrontUtilities.create();
+     *
+     *     Instant expirationDate = Instant.now().plus(7, ChronoUnit.DAYS);
```

Comment:
`plus(Duration.ofDays(7))`

Let's add a small blurb explaining what the example is illustrating. Ditto for the other examples

Reply:
Added small descriptions